pyproject.toml

```diff
@@ -42,3 +42,44 @@ ignore = [
 
 [tool.ruff.lint.flake8-quotes]
 avoid-escape = false
+
+[tool.ruff.format]
+exclude = [
```

Comment:
As we discussed offline. let's do gradual migration piece by piece: exludes for black and includes for ruff

Reply:
Done. This PR only migrates `python/ray/_private/` folder. I also updated the PR description.